editor/js/Menubar.File.js

```diff
@@ -248,6 +248,18 @@ Menubar.File = function ( editor ) {
 
 		} );
 
```

Comment:
Yeah... I guess we'll have to move that code back to `Menubar.File` and construct it as needed.